### PR TITLE
DOCS - Improve SEO

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Changed file locations to match urls
* Browser friendly, descriptive urls
* Added keywords, image and description meta tags

## Links to any relevant issues

* Resolves  iotaledger/docs.iota.org#34

## Type of change

- [ ] Enhancement (a non-breaking change which adds functionality)
- [ ] Documentation Fix

## How the change has been tested

Docs site was built and served locally using node 12.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@iotaledger/tech-writers  could someone review this please?

**Review by huhn511 — APPROVED** [review]
LGTM 👍

File: documentation/docs/specification.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+---
+description: The wallet.rs library is a stateful package with a standardized interface to build applications with IOTA value transactions.
+image: /img/logo/wallet_light.png
+keywords:
+- client library methods
```
Comment (Thoralf-M) [inline documentation/docs/specification.md]: Should this really be here?

File: documentation/docs/welcome.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+description: Official IOTA Wallet Library Software which can be used to easily integrate an IOTA Wallet into your application 
+image: /img/logo/wallet_light.png
+keywords:
+- requirements
+- wallet
+- software
+- library
+- rust
+- python
+- nodejs
```
Comment (Thoralf-M) [inline documentation/docs/welcome.md:11]:
```suggestion
- nodejs
- java
```

File: documentation/docs/libraries/overview.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
+---
+description: The `wallet.rs` library is written in Rust.  You can also find two bindings written in Node.js and Python.
```
Comment (Thoralf-M) [inline documentation/docs/libraries/overview.md]:
```suggestion
description: The `wallet.rs` library is written in Rust. You can also find three bindings written in Node.js, Python and Java.
```

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**